guides/plugins/plugins/testing/php-unit.md

```diff
@@ -196,72 +187,33 @@ cd ./../../../
 Now we can execute the `phpunit.sh` to run all tests of our plugin.
 
 ```text
-$ ./bin/phpunit.sh
+./bin/phpunit.sh
 ```
 
 ### Executing a single test
 
 If we want to execute a specific test only, we have to pass the path to the test as argument.
 
 ```text
```

Comment:
If you remove the leading `$` as indicator for a user/non-root command, wouldn't it be a good idea to mark these sections as shell scripts so highlighting suggest, what it could be instead?

Reply:
@JoshuaBehrens yes, they will be marked as ```shell in the future.

We discussed this in the team and the benefit of easy copying overweighed the indicator benefit. In 99% of the cases the usage is clear (bin/console etc.)

Reply:
I am with you. It is really annoying to remove these leading indicators :) just wanted to know, whether you will update the codefence hint as well